src/ControllerBlocks.f90

```diff
@@ -277,7 +277,7 @@ REAL FUNCTION PitchSaturation(LocalVar, CntrPar, objInst, DebugVar)
         TYPE(DebugVariables), INTENT(INOUT)     :: DebugVar
 
         ! Define minimum blade pitch angle as a function of estimated wind speed
-        PitchSaturation = interp1d(CntrPar%PS_WindSpeeds, CntrPar%PS_BldPitchMin, LocalVar%WE_Vw_F)
+        PitchSaturation = interp1d(CntrPar%PS_WindSpeeds, CntrPar%PS_BldPitchMin, LocalVar%WE_Vw)
```

Comment:
This could have a large effect on loads and power capture.  Why was it added?

Reply:
This wasn't supposed to be added - removed!